.vscode/settings.json

```diff
@@ -3,7 +3,7 @@
 	"eslint.experimental.useFlatConfig": true,
 	"eslint.format.enable": true,
 	"[typescript]": {
-		"editor.defaultFormatter": "biomejs.biome"
+		"editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
what's the reason for this? We are switching to biome for formatting. I might have to update the docs on this

Reply:
Sorry, I forgot to delete it